src/Cupboard.Providers.Windows/Chocolatey/ChocolateyPackageProvider.cs

```diff
@@ -28,12 +28,34 @@ public override ChocolateyPackage Create(string name)
 
         protected override bool IsPackageInstalled(ChocolateyPackage resource, string output)
         {
-            return output.Contains(resource.Package, StringComparison.OrdinalIgnoreCase);
+            var containsPackage = output.Contains(resource.Package, StringComparison.OrdinalIgnoreCase);
+            if (string.IsNullOrEmpty(resource.PackageVersion))
+            {
+                return containsPackage;
+            }
+
+            if (!containsPackage)
+            {
+                return false;
+            }
+
+            var indexOfPipe = output.IndexOf('|');
+            var indexOfNewLine = output.IndexOfAny(new[] { '\r', '\n' }, indexOfPipe);
+            if (indexOfPipe == -1 || indexOfNewLine == -1)
+            {
+                return false;
+            }
```

Comment:
If I understand correctly, this will work with the first item in the list and won't work if a package contains multiple packages such as `uplay` (see picture):

![image](https://user-images.githubusercontent.com/357872/134250102-1a6c8a4e-1771-411d-8ec2-559714d62f09.png)

Reply:
Right, `--exact` would be useful here.